catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -724,7 +724,7 @@ private void addToFieldsCache(Set<AttributeDescriptor> descriptors) {
 
       fieldsCache.add(ad.getName() + schemaFields.getFieldSuffix(format));
 
-      if (!getSpecialIndexSuffix(format).equals("")) {
+      if (!"".equals(getSpecialIndexSuffix(format))) {
```

Comment:
✏️ This is unnecessary; a null format would result in an NPE on line 694 in the stack chain of the call to `getSpecialIndexSuffix`.